docs/source/tracking/tracking-api.rst

```diff
@@ -254,20 +254,49 @@ MLflow also supports running multiple runs in parallel using `multiprocessing <h
 
 .. code-block:: python
 
-        import mlflow
-        import multiprocessing as mp
+    import mlflow
+    import multiprocessing as mp
 
 
-        def train_model(params):
-            with mlflow.start_run():
-                mlflow.log_param("p", params)
-                ...
+    def train_model(params):
+        with mlflow.start_run():
+            mlflow.log_param("p", params)
```

Comment:
```suggestion
            mlflow.log_params(params)
```

this line looks a bit strange. Can we use `log_params`?

Reply:
Ah true, good catch!!